test/stdlib/TestDecimal.swift

```diff
@@ -237,6 +237,10 @@ class TestDecimal : TestDecimalSuper {
                 expectEqual(Decimal(i*j), Decimal(i) * Decimal(j), "\(Decimal(i*j)) == \(i) * \(j)")
                 expectEqual(Decimal(i+j), Decimal(i) + Decimal(j), "\(Decimal(i+j)) == \(i)+\(j)")
                 expectEqual(Decimal(i-j), Decimal(i) - Decimal(j), "\(Decimal(i-j)) == \(i)-\(j)")
+                // Strideable methods
+                expectEqual(Decimal(i).advanced(by: Decimal(j)), Decimal(i+j), "\(Decimal(i).advanced(by: Decimal(j))) == \(i) + \(j)")
+                expectEqual(Decimal(i).distance(to: Decimal(j)), Decimal(j-i), "\(Decimal(i).distance(to: Decimal(j))) == \(j) - \(i)")
+                expectEqual(Decimal(i).advanced(by: Decimal(i)).distance(to: Decimal(j)), Decimal(j), "Decimal(i).advanced(by: Decimal(i).distance(to: Decimal(j)) == Decimal(j)")
```

Comment:
Still missing a paren within your string :)
And if you're following the style of earlier lines, consider escaping appropriate portions of each side of the equation in the string with `\()`.

Reply:
This currently looks like it's computing `j - (i + i)`, but maybe it should be `(i + j) - i`:

```swift
Decimal(i).distance(to: Decimal(i).advanced(by: Decimal(j)))
```

Or `i + (j - i)`:

```swift
Decimal(i).advanced(by: Decimal(i).distance(to: Decimal(j)))
```